ci/travis/install-bazel.sh

```diff
@@ -113,16 +103,16 @@ EOF
     fi
   fi
   if [ 0 -ne "${upload}" ]; then
-    translated_path="${HOME}/bazel_cache_credential.json"
+    translated_path=~/bazel_cache_credential.json
     if [ "${OSTYPE}" = msys ]; then  # On Windows, we need path translation
       translated_path="$(cygpath -m -- "${translated_path}")"
     fi
-    cat <<EOF >> "${HOME}/.bazelrc"
+    cat <<EOF >> ~/.bazelrc
 build --google_credentials="${translated_path}"
 EOF
   else
     echo "Using remote build cache in read-only mode." 1>&2
-    cat <<EOF >> "${HOME}/.bazelrc"
+    cat <<EOF >> ~/.bazelrc
```

Comment:
what's difference between `${HOME}` and `~`?

Reply:
Nothing of significance here; I just made that change because it's easier to read and gets rid of unnecessary quotes. But in theory, `HOME` is an environment variable that can be easily modified to point to anything, whereas `~` is guaranteed to expand to the home directory.